docs/source/channel_update_tutorial.rst

```diff
@@ -239,16 +205,19 @@ Issue the following commands to operate as the Org1 admin.
 
   # you can issue all of these commands at once
 
+  export FABRIC_CFG_PATH=${PWD}/../config/
```

Comment:
You also need to add the peer cli now to your path for the following command:

```
export PATH=${PWD}/../bin:$PATH
```

Reply:
Done